app/styleguide/page.tsx

```diff
@@ -51,22 +50,18 @@ import {
   CardMutedText,
 } from "@/components/Card/card";
 
-import {
-  Example,
-  BrandColor,
-  handleDelete,
-} from "./sg-components";
+import {BrandColor, Example, handleDelete,} from "./sg-components";
```

Comment:
@fraserclark is your editor doing automatic formatting on save? It's collapsing this import line that was deliberately spread over multiple lines.

Reply:
@fraserclark and @andrewebdev we should probably decide on some standard formatting and make sure all of our formatters are configured the same way. Are we all using VSCode? Maybe we can share a configuration file or something.

Reply:
I'm using PhpStorm - because im old and use it daily! I tried VS code a few times including on DMP phase 1 stuff - but just couldn't quite get use to it so back to my old familar.

Yeah PhpStorm does a lot of auto-formatting out the box but we can set up VS Code and PhpStorm both work with .prettierrc, eslintrc and editorconfig so can probably sync all 3 update.

I'd defer to you guys on setting up the "standards" for these 3 files (Think we already 2 of the 3), I have no strong opinions on any formatting stuff.

Reply:
I use a combination of Emacs and NeoVim, so no common config. This is where EditorConfig is helpful. But I would always advocate for **less** auto-formatting. Editor config takes care of important stuff, like making sure everyone uses the same line-endings. But outside that auto-formatters makes too many assumptions about the code and often gets in the way, like for example the code blocks mentioned in my other comments.